CMakeLists.txt

```diff
@@ -163,7 +172,7 @@ if ( WIN32 )
     set ( version ${HPCC_MAJOR}.${HPCC_MINOR}.${HPCC_POINT} )
     set ( CPACK_PACKAGE_INSTALL_DIRECTORY "${DIR_NAME}\\\\${version}\\\\eclide" )
     ####  Change the following if you _don't_ want the installer to uninstall previous versions ####
-    set ( CPACK_PACKAGE_INSTALL_REGISTRY_KEY "eclide_${HPCC_MAJOR}.0.0" )
+    set ( CPACK_PACKAGE_INSTALL_REGISTRY_KEY "eclide_${version}" )

```

Comment:
Is this correct?  The install strategy is that there should only be one IDE installed per major version, so if I install 6.4 it would replace 6.2.x).

The same is not true for the client tools.